LAKMobile/src/screens/AddJob.tsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
```

Comment:
This is what came up for me on the android simulator. Does the same thing happen on your simulator? @shreekarp
Might have to fix the CSS so that it matches the figma on the emulator if that's the case.

<img width="376" alt="image" src="https://user-images.githubusercontent.com/56745367/156122472-cb6cdf8e-7d51-4108-b276-466c515035bf.png">